src/main/kotlin/no/nav/syfo/consumer/azuread/v2/AzureAdV2TokenInvoker.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.syfo.consumer.azuread.v2
+
+import org.springframework.beans.factory.annotation.*
+import org.springframework.http.*
+import org.springframework.stereotype.Component
+import org.springframework.util.MultiValueMap
+import org.springframework.web.client.RestTemplate
+
+@Component
+class AzureAdV2TokenInvoker @Autowired constructor(
```

Comment:
Måtte ta ut denne som en egen komponent for å få til enhetstestene av AzureAdV2TokenConsumer

Reply:
Med Spring sin MockRestServiceServer så slipper man å gjøre dette for å mocke nærmest mulig RestTemplate-kallet. Kan bruke den i stedet for å mocke selve serveren i stedet for mockito her.
Er et eksempel på det med AzureADV2Mock.kt i Syfobehandlendenhet. Må evt. tilpasses litt, men er ca. det som trengs for å mocke AD-server-kall.

Reply:
Ja, ser at det du foreslår er en mulighet. Men jeg synes ikke det er så stor ulempe å splitte den ene komponenten i to. I dette tilfellet er det også praktisk å bruke Mockito siden det er enkelt å få til en mock som returnerer et token i den første invokeringen og et annet token i den andre invokeringen.